src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```diff
@@ -35,6 +35,17 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ProjectCapability Include="AspNetCoreInProcessHosting" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <!--
+      Unless the project explicitly set the property value to 'true' (case-insensitive),
+      default it to 'false'. Not specifying a value at all (or specifying an empty value)
+      will also default to 'false'. In .NET 5.0 this functionality is explicitly disabled
+      by default for web project types, and in future .NET releases the "disable by default"
+      logic will roll out to other project types.
+    -->
+    <EnableUnsafeBinaryFormatterSerialization Condition="'$(EnableUnsafeBinaryFormatterSerialization)' != 'true'">false</EnableUnsafeBinaryFormatterSerialization>
```

Comment:
How easily does this break an existing application? For example, if I put images in a `.resx` file is my app broken now?

Reply:
Correct, such an app will be broken. However:

* The overwhelming majority of web applications don't do such a thing; and
* The exception message text links to a document with clear remediation steps.